generators/generator-microsoft-bot-adaptive/generators/app/templates/assets/botName.botproj

```diff
@@ -1,5 +1,5 @@
 {
   "$schema": "https://raw.githubusercontent.com/microsoft/BotFramework-Composer/main/Composer/packages/server/schemas/botproject.schema",
-  "name": "who",
+  "name": "",
```

Comment:
Should this be `botName`?

Reply:
In this case, no. What happens here is we read the JSON in and convert to an object, as we need to populate skills settings potentially. We use this opportunity to just stamp the name in to the object before writing it back out to disk.